src/CMakeLists.txt

```diff
@@ -29,6 +29,9 @@ endif()
 
 set(CMAKE_FIND_FRAMEWORK LAST)
 
+# Helper utility for overriding cached variables
+include(UpdateCacheVariable)
```

Comment:
Moved this up here because this command is used in more than just the OpenMP block.